### ISSUE TITLE
Crashing

### ISSUE THREAD

**Issue by nadimalam** [dmratcliffe/python-greenscreen#1]
Hi,

Thanks a lot for this tool, but its constantly crashing when I pick the Kinect V2. Any idea if im missing something? I just downloaded the main.exe executable file and am using that.

Thanks

**Comment by dmratcliffe** [comment]
I'm not sure how this happens and it's hard for me to debug because my test systems don't have this isusue. The main problem is the windows dialogue isn't actually supposed to appear, I have no code that does that anyway. A restart usually fixes it. I'll write something into the readme about it.

**Comment by nadimalam** [comment]
So i did try with a restart also, but still same issue. I get the popup to choose the camera source and after i choose the kinect v2 and click Okay, the whole app crashes.

Just want to make sure of the folder structure, its just the Main.exe file thats required right? and it can sit in any folder?

**Comment by dmratcliffe** [comment]
yes, it can sit anywhere. The strange thing is that popup isn't a part of my code. Try right clicking and running as admin? You extracted it right?

**Comment by dmratcliffe** [comment]
Hey! I did some testing, it seems that the "Video Source" dialog appears when the camera is already in use by another application. Try making sure that the camera isn't being used by anything!

**Comment by dmratcliffe** [comment]
any update?

**Comment by nadimalam** [comment]
Hey, sorry i haven't been able to test again. Hopefully i will do soon. But im pretty sure last time i checked straight after a PC restart so the camera wouldn't have been open or used by anything else.

**Comment by dmratcliffe** [comment]
Yeah, I had another person try. It appears its an issue with my app and the kinectv2. Not sure whats wrong.

**Comment by nadimalam** [comment]
ok no worries
